lib/model/token.dart

```diff
@@ -56,6 +57,7 @@ class Token {
         ? map["expires_in"]
         : int.tryParse(map["expires_in"].toString()) ?? 60;
     model.refreshToken = map["refresh_token"];
+    model.idToken = map.containsKey("id_token") ? map["id_token"] : '';
```

Comment:
I'd add it as well to `toJsonMap` for completeness

Reply:
done